libs/platform/src/lib/components/search-field/search-field.component.html

```diff
@@ -1,106 +1,62 @@
-<div
-    class="search-field"
-    [ngClass]="{
+<div class="search-field" [ngClass]="{
         'with-categories': showCategoryDropdown,
         'is-loading': isLoading,
         'hide-category-label': hideCategoryLabel
-    }"
->
-    <div
-        class="search-field--input-group fd-input-group"
-        [attr.aria-controls]="menuId"
-        [attr.aria-expanded]="showDropdown ? 'true' : 'false'"
-        aria-haspopup="true"
-        #inputGroup
-    >
-        <span
-            class="fd-input-group__addon fd-input-group__addon--button"
-            [ngClass]="{ 'fd-input-group__addon--compact': compact }"
-            *ngIf="showCategoryDropdown"
-        >
-            <button
-                class="search-field--category-button fd-input-group__button fd-button fd-button--menu"
-                [ngClass]="{ 'fd-button--compact': compact }"
-                [fdpMenuTriggerFor]="categoryMenu"
-            >
-                <span class="search-field--category-label" *ngIf="!hideCategoryLabel">{{
+    }">
+    <div class="fdp-search-field__input-group fd-input-group" [attr.aria-controls]="menuId"
+        [attr.aria-expanded]="showDropdown ? 'true' : 'false'" aria-haspopup="true" #inputGroup>
+        <span class="fd-input-group__addon fd-input-group__addon--button"
+            [ngClass]="{ 'fd-input-group__addon--compact': compact }" *ngIf="showCategoryDropdown">
+            <button class="fdp-search-field__category-button fd-input-group__button fd-button fd-button--menu"
+                [ngClass]="{ 'fd-button--compact': compact }" [fdpMenuTriggerFor]="categoryMenu">
+                <span class="fdp-search-field__category-label" *ngIf="!hideCategoryLabel">{{
                     currentCategory ? currentCategory.label : categoryLabel
                 }}</span>
             </button>
         </span>
-        <input
-            #inputField
-            type="text"
-            placeholder="{{ placeholder }}"
-            class="search-field--input fd-input fd-input-group__input"
-            [attr.id]="inputId"
-            [attr.disabled]="disabled ? '' : null"
-            [ngClass]="{ 'fd-input--compact': compact }"
-            (keydown)="onKeydown($event)"
-            (keydown.enter)="onSearchSubmit()"
-            [(ngModel)]="inputText"
-            (ngModelChange)="onValueChange($event)"
-        />
-        <span
-            class="fd-input-group__addon fd-input-group__addon--button"
-            [ngClass]="{ 'fd-input-group__addon--compact': compact }"
-            *ngIf="inputText && inputText.length > 0"
-        >
+        <input #inputField type="search" placeholder="{{ placeholder }}"
```

Comment:
Let's try to use `[attr.placeholder]` syntax over interpolation, unless there is a specific reason for it

Reply:
I've changed it to `[attr.placeholder]`.